src/Shared/MSBuildNameIgnoreCaseComparer.cs

```diff
@@ -22,16 +22,7 @@ internal class MSBuildNameIgnoreCaseComparer : IConstrainedEqualityComparer<stri
         /// <summary>
         /// The processor architecture on which we are running, but default it will be x86
         /// </summary>
-        private static readonly NativeMethodsShared.ProcessorArchitectures s_runningProcessorArchitecture;
-
-        /// <summary>
-        /// We need a static constructor to retrieve the running ProcessorArchitecture that way we can
```

Comment:
`We need a static constructor` did I overlook this? Is the static constructor needed to avoid issues on IA64?

Reply:
I don't think the comment was specifically about needing an explicit static ctor. Switching to inline initialization should be fine.